system/core/base.php

```diff
@@ -459,6 +459,24 @@ function hypha_setUser($user, $username, $password, $fullname, $email, $language
 		}
 	}
 
+	/*
+		Function: hypha_retireUser
+		Retires the given user and adds a note to the digest about this.
+	*/
+	function hypha_retireUser($user, $by_admin) {
```

Comment:
Passing the user_id in stead of the user object is a bit more clear and less chances on passing the wrong user object, IMO.

Reply:
Makes sense. Does mean the user must be looked up twice in one of the instances, but that's not really a big deal. Fixed!